bootstrap.sh

```diff
@@ -7,7 +7,7 @@
 ########################################################################################################################
 
 
-source ./.dd4hep-ci.d/init_x86_64.sh
+source /cvmfs/sft.cern.ch/lcg/views/LCG_101/x86_64-centos7-gcc11-opt/setup.sh
```

Comment:
I think this requires changes to `Geant4_DIR=$G4INSTALL/lib/Geant4-10.3.3` below.
And won't work for different Operating Systems.

Reply:
This would fix all software versions to what is defined by the LCG view.
I am not sure this is what is always desired.

Reply:
Is there alternative way to setup required environment?

on CC7 both Geant4-10.3.3 and Geant4-10.7.2 work

Reply:
@MarkusFrankATcernch the `init_x...` was always just forwarding to LCG builds, (or other CVMFS provided packages)
so fixing it to a specific one would be fine, so that the bootstrap script is at least internally consistent.

@kjvbrt looking at our CI script, Geant4_DIR isn't needed at all

https://github.com/AIDASoft/DD4hep/blob/fa17ba1e91fac792842d2b499d9eb5222c0718a9/.github/scripts/compile_and_test.sh#L13-L28

Reply:
Yes, script finishes correctly without it